src/dbg/debugger.cpp

```diff
@@ -127,7 +128,7 @@ static DWORD WINAPI memMapThread(void* ptr)
             memMapThreadCounter = GetTickCount();
         }
 
-        Sleep(50);
+        Sleep(1000);

```

Comment:
I agree something like TaskThread might be nice here. I didn't realize memMapThreadCounter was externally visible and reset, so I mistakenly thought that sleep(1000) wouldn't change any behavior. 

Now that I have a better understanding of the code, I'll replace it with a TaskThread which has the desired semantics.